test/viewmodels/checkAnswers/sections/transportUnit/TransportSealChoiceSummarySpec.scala

```diff
@@ -0,0 +1,93 @@
+
+import base.SpecBase
+import controllers.sections.transportUnit.{routes => transportUnitRoutes}
+import fixtures.TransportUnitFixtures
+import fixtures.messages.sections.transportUnit.TransportSealChoiceMessages
+import models.CheckMode
+import org.scalatest.matchers.must.Matchers
+import pages.sections.transportUnit.TransportSealChoicePage
+import play.api.i18n.Messages
+import play.api.test.FakeRequest
+import uk.gov.hmrc.govukfrontend.views.Aliases.Value
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+class TransportSealChoiceSummarySpec extends SpecBase with Matchers with TransportUnitFixtures {
+
+  "TransportSealChoiceSummary" - {
+
+    lazy val app = applicationBuilder().build()
+
+    Seq(TransportSealChoiceMessages.English).foreach { messagesForLanguage =>
+
+      s"when being rendered in lang code of '${messagesForLanguage.lang.code}'" - {
+
+        implicit lazy val msgs: Messages = messages(app, messagesForLanguage.lang)
+
+        "when there's no answer" - {
+
+          "must output a row with not provided" in {
+
+            implicit lazy val request = dataRequest(FakeRequest(), emptyUserAnswers)
+
+            TransportSealChoiceSummary.row(testIndex1) mustBe
+              Some(
+                SummaryListRowViewModel(
+                  key = messagesForLanguage.cyaLabel,
+                  value = Value(Text(messagesForLanguage.notProvided)),
+                  actions = Seq(
+                    ActionItemViewModel(
+                      content = messagesForLanguage.change,
+                      href = transportUnitRoutes.TransportSealChoiceController.onPageLoad(testErn, testDraftId, testIndex1, CheckMode).url,
+                      id = "changeTransportSealChoice1"
+                    ).withVisuallyHiddenText(messagesForLanguage.moreInfoCyaChangeHidden)
+                  )
+                )
+              )
+          }
+        }
+
+        "when there's an answer" - {
+
+          s"must output the expected row for TransportSealChoice" in {
+
+            implicit lazy val request = dataRequest(FakeRequest(), emptyUserAnswers.set(TransportSealChoicePage(testIndex1), true))
+
+            TransportSealChoiceSummary.row(testIndex1) mustBe
+              Some(
+                SummaryListRowViewModel(
+                  key = messagesForLanguage.cyaLabel,
+                  value = Value(Text("Yes")),
+                  actions = Seq(
+                    ActionItemViewModel(
+                      content = messagesForLanguage.change,
+                      href = transportUnitRoutes.TransportSealChoiceController.onPageLoad(testErn, testDraftId, testIndex1, CheckMode).url,
+                      id = "changeTransportSealChoice1"
+                    ).withVisuallyHiddenText(messagesForLanguage.moreInfoCyaChangeHidden)
+                  )
+                )
+              )
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Can you add a newline to the end of the last line of this file? The reason Git, GitHub, and others make a fuss is that according to the Unix standard all lines in a text file must be terminated by a newline, and you can encounter all sorts of problems with standard Unix tools if a file doesn't

You can make IntelliJ do this automatically to every file you edit by setting "Ensure every saved file ends with a line break"